libs/timer/throttle_timer_test.go

```diff
@@ -67,14 +67,16 @@ func TestThrottle(test *testing.T) {
 	time.Sleep(longwait)
 	assert.Equal(2, c.Count())
 
-	// send 12, over 2 delay sections, adds 3
+	// send 12, over 2 delay sections, adds 3 or more. It
```

Comment:
Allowed timer to be set a higher number of times in timer throttle test to account for slow execution of loop

--- FAIL: TestThrottle (0.81s)
    throttle_timer_test.go:76: 
        	Error Trace:	throttle_timer_test.go:76
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 6
        	Test:       	TestThrottle
FAIL
FAIL	github.com/tendermint/tendermint/libs/common	8.109s